autopayroll-CordaService/build.gradle

```diff
@@ -0,0 +1,157 @@
+buildscript {
+    Properties constants = new Properties()
+    file("$projectDir/../constants.properties").withInputStream { constants.load(it) }
+
+    ext {
+
+        corda_release_group = constants.getProperty("cordaReleaseGroup")
+        corda_release_version = constants.getProperty("cordaVersion")
+        corda_gradle_plugins_version = constants.getProperty("gradlePluginsVersion")
+        kotlin_version = constants.getProperty("kotlinVersion")
+        junit_version = constants.getProperty("junitVersion")
+        quasar_version = constants.getProperty("quasarVersion")
+        log4j_version = constants.getProperty("log4jVersion")
+        slf4j_version = constants.getProperty("slf4jVersion")
+        corda_platform_version = constants.getProperty("platformVersion").toInteger()
+    }
+
+    repositories {
+        mavenLocal()
```

Comment:
This is intentional for developers who will try out these sample behind a firewall. They will compile their corda at their local repository and these corDapp will build use the local repositories.

Reply:
That doesn't make sense. If the firewall is blocking mavenCentral then they won't be able to compile corda either.

Reply:
From our experience, we have had worked with many enterprise developers behind the firewall. It is the solution for them to proceed with these samples

Reply:
It still doesn't make sense. `mavenLocal()` refers to your `<HOME>/.m2/repositories` directory. In order to use that behind a firewall someone would have to zip up a copy of a working one and unzip it. What is more likely is that the company has their own repo that proxies out to maven central. If you still think `mavenLocal()` makes sense because of firewall restrictions then you should write something in the readme explaining the steps on how to compile behind a firewall.